src/pages/Credentials/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { Link, useNavigate, useParams } from 'react-router-dom';
+import { Alert, ListGroup } from 'react-bootstrap';
+import Card from 'src/components/Card';
+import Icon from 'src/components/Icon';
+import CredentialCard from 'src/containers/CredentialCard';
+// import sampleAvatar from 'src/assets/images/sample-avatar.png';
+import { useAppContext } from 'src/store';
+import { beautifyText } from 'src/utilities';
+import styles from './index.module.scss';
+import cn from 'classnames';
+
+function Credentials() {
+  const navigate = useNavigate();
+  const { state } = useAppContext();
+  const { credentials, verification } = state || {};
+  const { id } = useParams();
+
+  const renderPartialDataCard = (claim, id: string | number, isClickable?: boolean) => {
+    const isKyc = 'type' in claim;
```

Comment:
`claim.type === 'verification'` correct condition

Reply:
honestly, it's confusing for me because i your condition (` {field === 'type' ? 'Know Your Customer (KYC)' : claim[field]}`).
should I change ur condition too or is it related to something else?

Reply:
yes cause other VC has no type I'd condition like this but I want to sent type on experience too as well so this condition would be more proper but on that loop I just check the field key anyway I can fix it my self later keep going ....